### PR TITLE
Creating required diagrams

### PR DESCRIPTION
**Pull request by Palingenae**
This PR is about the creation of the following diagrams:
- [x] **Use Case:** Comment creation from visitor
- [x] **Use Case:** Comment validation from administrator
- [x] **Sequence:** Writing a new blog post
- [x] **Sequence;** Creating a new user
- [x] **Sequence;** Creating comments
- [x] **Sequence:** Validating comments
- [x] **Class** 
- [x] **Entity-Relationships**

### CODE REVIEW

File: doc/diagrams/Classes.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            -string $nickname
+            -string $mail_address
+            -string $password
+            -string $role
+            -Article[] $articles
+        }
+        %% 1 -> 0..n
+        User "1" --> "0..*" Article : redacted
+        %% 0..1 -> 0..n
+        User "1..*" --> "0..*" Comment : wrote
```
Comment (gth076) [inline doc/diagrams/Classes.md:16]: If I doesn't make a mistake on this relation one comment can be related to one or several users ?
I think a comment can only be related to the user who wrote it ?

**Review by gth076 — COMMENTED** [review]
I just wrote a comment on the class diagramm (which is also for the entity-relation diagram) I think the relation between User and Comment might be 1 --> 0..* 
If a comment is written it is related to only one user ??

File: doc/diagrams/Classes.md (hunk 1 above)
Comment (Palingenae) [reply doc/diagrams/Classes.md:16]: I did make a mistake. The comment is right ; an user could not write a comment or be the only one to write it. I'll correct that.

**Review by gth076 — APPROVED** [review]
Diagrams modified as suggested

**Pull request merged**